conjure/steps/01_get-kubectl.sh

```diff
@@ -1,26 +1,15 @@
 #!/bin/bash
 #
-# Description: Download kubectl for controlling Kubernetes Cluster
+# Title: Get kubectl for controlling a Kubernetes Cluster
 
 . /usr/share/conjure-up/hooklib/common.sh
 
 KUBECTL_PATH=$HOME/conjure-up/kubernetes
 mkdir -p $KUBECTL_PATH || true
 
-if [ $(unitStatus kubernetes 0) = "error" ]; then
-    exposeResult "There is an error with the kubernetes service, plese check juju status." 1 "false"
-fi
-
-if [ $(unitStatus etcd 0) = "error" ]; then
-    exposeResult "There is an error with the etcd service, plese check juju status." 1 "false"
-fi
-
-if [ $(unitStatus kubernetes 0) != "active" ]; then
-    exposeResult "Kubernetes is not quite ready yet" 0 "false"
-fi
-
 # TODO: Convert to an actionable item
-juju scp kubernetes/0:kubectl_package.tar.gz $KUBECTL_PATH/.
+debug "Found leader: $(getLeader kubernetes)"
+juju scp $(getLeader kubernetes):kubectl_package.tar.gz $KUBECTL_PATH/.
```

Comment:
> I don't see the getLeader function in this pull request but I imagine it is doing a juju run service is_leader which is a time expensive operation. I would recommend saving the leader to a variable and using that instead of calling getLeader twice.

Thanks I'll get this fixed up